internal-registry/eclipse/cloud-shell/nightly/devworkspacetemplate.yaml

```diff
@@ -0,0 +1,26 @@
+kind: DevWorkspaceTemplate
+apiVersion: workspace.devfile.io/v1alpha2
+metadata:
+  name: cloud-shell
+spec:
+  components:
+    - name: web-terminal
+      container:
+        image: "${RELATED_IMAGE_plugin_eclipse_cloud_shell_nightly}"
+        mountSources: false
+        command: ["/go/bin/che-machine-exec",
+                  "--authenticated-user-id", "$(DEVWORKSPACE_CREATOR)",
+                  "--idle-timeout", "$(DEVWORKSPACE_IDLE_TIMEOUT)",
+                  "--pod-selector", "controller.devfile.io/workspace_id=$(CHE_WORKSPACE_ID)",
```

Comment:
let's start using DEVWORKSPACE_ID here and in other place instead of Che related

Reply:
:+1: